src/recipes/berrylicious-frozen-sangria-slush.json

```diff
@@ -0,0 +1,56 @@
+{
+    "name": "Berrylicious Frozen Sangria Slush",
+    "description": "This boozy, fruity frozen sangria is an adult version of the icy slushies we slurped down as kids. So refreshing on a hot summer night. Cheers!",
+    "github": "wfebriad",
+    "ingredients": [
+        {
+            "quantity": "1",
+            "measure": "",
+            "ingredient": "1 cup red wine"
```

Comment:
Quantity and measure can go into their respective fields, instead of in the ingredients field.

Reply:
ok, i will fixed it, thank you